### PR TITLE
[CI:docs] hack/podmansnoop: print arguments

### PR DESCRIPTION
**Pull request by vrothberg**
Update the snoop script to also include the arguments to make the output more useful.

```
$ sudo hack/podmansnoop
PCOMM            PID     PPID    AGE(ms) ARGV
conmon           14964   14952   1.01    /usr/bin/conmon --version
podman           14952   14139   26.07   /usr/bin/podman ps
```

Signed-off-by: Valentin Rothberg <vrothberg@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

@rhatdan PTAL

### CODE REVIEW

**Comment by vrothberg** [comment]
@cevich the "Windows Smoke Test" and "Verify Win Installer Build" seem too much for a docs PR. Is that on purpose?

**Comment by vrothberg** [comment]
repushed since CI seemed stuck

File: hack/podmansnoop [hunk 1]
```diff
@@ -42,41 +43,112 @@ TRACEPOINT_PROBE(sched, sched_process_exit)
...
+        except Exception:
+            pass
         return
 
     age = (e.exit_time - e.start_time) / 1e6
-    print("%-16s %-7d %-7d %-7d %-7.2f " %
-              (task, e.pid, e.ppid, e.tid, age), end="")
+    argv = b' '.join(pidToArgv[e.pid]).replace(b'\n', b'\\n')
+    print("%-16s %-7d %-7d %-7.2f %s" %
+              (comm, e.pid, e.ppid, age, argv), end="")
```
Comment (Luap99) [inline hack/podmansnoop:144]: This seems to print the argv as `b'command with args'`, can you remove the unnecessary `b''`?

**Comment by cevich** [comment]
> @cevich the "Windows Smoke Test" and "Verify Win Installer Build" seem too much for a docs PR. Is that on purpose?

Oddly enough, we have to run these because podman needs to be compiled in order to verify documentation.  Then the docs are also included in the Windows package, so we need to make sure that's working.  The exception is the Windows Smoke Test, I think we can exclude that one from `[CI:DOCS]`, I'll toss up a PR.

**Comment by cevich** [comment]
Opened https://github.com/containers/podman/pull/16680

File: hack/podmansnoop [hunk 2]
```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env python
 from __future__ import print_function
 
-# Based on the `exitsnoop` scripts at https://github.com/iovisor/bcc/blob/master/tools/exitsnoop.py
+# Based on the `exitsnoop` script at https://github.com/iovisor/bcc/blob/master/tools/exitsnoop.py
+# Based on the `execsnoop` script at https://github.com/iovisor/bcc/blob/master/tools/execsnoop.py
```
Comment (Luap99) [inline hack/podmansnoop:5]: duplicates lines?

**Review by Luap99 — COMMENTED** [review]
sorry missed this yesterday,  otherwise LGTM

**Comment by vrothberg** [comment]
Almost, one is exit the other exec. I copied from both tools.

**Pull request merged**